### ISSUE TITLE
class Tuplet should allow bracket and placement to be None, meaning "not specified"

### ISSUE THREAD

**Issue by gregchapman-dev** [cuthbertLab/music21#1398]
**music21 version**

`8.0.0a12`   

**Problem summary**

tuplet.placement = None and tuplet.bracket = None are valid setting (and in fact should be the default).

`tuplet.placement is None` should be handled in writers by not specifying placement in the output file (and that is, in fact, what m21ToXml.py does).
`tuplet.bracket is None` should be handled the same way (but m21ToXml.py doesn't handle it properly; None is treated as False).

My Humdrum parser tries to carefully set up tuplet.placement and tuplet.bracket to match what the Humdrum file does (or does not) specify, defaulting both tuplet.placement and tuplet.bracket to None.  The new type-hinting in duration.Tuplet makes this impossible.

**Comment by gregchapman-dev** [comment]
To be more concise: MusicXML allows both tuplet placement and tuplet bracket to be unspecified.  So should music21 (and it did until very recently).

**Comment by mscuthbert** [comment]
yeah, that's probably a good idea.  I'm trying to get away from our old system of "None" = unknown or unspecified, and will probably go with a StringEnum giving UNSPECIFIED explicitly, because "None" sounds like "none" (no bracket should be written).
